src/main/java/com/internet/shop/model/User.java

```diff
@@ -15,11 +16,20 @@ public User(String name, String login, String password) {
         this.password = password;
     }
 
-    public User(Long id, String name, String login, String password) {
+    public User(Long id, String name, String login, String password, byte[] salt) {
         this.userId = id;
         this.name = name;
         this.login = login;
         this.password = password;
+        this.salt = salt;
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/50640794/94657939-d602dc00-030a-11eb-8d0d-fff1528770b1.png)
let's modify these constructors like this

Reply:
wow, thanks.
never faced this way declaration before)